data/README.md

```diff
@@ -387,3 +388,40 @@ To check conformance of a `BYTE_STREAM_SPLIT` decoder, read each
 `BYTE_STREAM_SPLIT`-encoded column and compare the decoded values against
 the values from the corresponding `PLAIN`-encoded column. The values should
 be equal.
+
+## Hive Map Schema
```

Comment:
Why "Hive"?

Reply:
This was due to schema being labeled with `message hive_schema {...`, but if anyone's searching for this keyword, they should find it below. I'll rename this section to match the filename.